python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -649,3 +649,101 @@ def predict(self, x: Tensor):
             )
 
         return detections
+
+# Unlike the Pytorch wrapper, this wrapper does not inherit from
```

Comment:
What if we push these comments to the docstring below?

Reply:
fixed